ci/build/arch-override.json

```diff
@@ -0,0 +1,8 @@
+{
+  "rpm": {
+    "armv7l": "armhfp"
+  },
+  "deb": {
+    "armv7l": "armhf"
```

Comment:
I worry about adding an extra file just because it seems like more to maintain. Also the json means we can't really put comments to explain why we have this. Any thoughts on writing a function to pass in `rpm` or `deb` and then returning values instead?

Reply:
My reasoning to put this in a separate file is that its easier to add overrides for other architectures should we need it in the future (say powerpc/s390x) and that it also separates code (building packages) from data (what overrides to apply when)

Reply:
Ah, okay that makes sense then :) Let's leave as is!